src/features/pages/Freeplay/components/TopBar.tsx

```diff
@@ -2,27 +2,34 @@ import React, { MouseEvent } from 'react'
 import { Select } from '@/components'
 import { formatInstrumentName } from '@/utils'
 import { gmInstruments, InstrumentName } from '@/features/synth'
-import { ArrowLeft, Midi } from '@/icons'
+import { ArrowLeft, Midi, StartRecord, StopRecord } from '@/icons'
 import { ButtonWithTooltip } from '../../PlaySong/components/TopBar'
 import Link from 'next/link'
 
 type TopBarProps = {
   isError: boolean
   isLoading: boolean
+  isRecordingAudio: boolean
   value: InstrumentName
   onChange: (instrument: InstrumentName) => void
   onClickMidi: (e: MouseEvent<any>) => void
+  onClickRecord: (e: MouseEvent<any>) => void
 }
 
-export default function TopBar({ isError, isLoading, value, onChange, onClickMidi }: TopBarProps) {
+export default function TopBar({ isError, isLoading, isRecordingAudio, value, onChange, onClickMidi, onClickRecord }: TopBarProps) {
+  const recordTooltip = isRecordingAudio? "Save record" : "Start recording audio";
+  
   return (
     <div className="px-4 text-white transition text-2xl h-[50px] min-h-[50px] w-full bg-[#292929] flex items-center gap-4">
```

Comment:
It's feeling a little cramped, could you increase the gap?

Reply:
What you mean?

Reply:
I mean that the buttons are very close together and I think it'd look better with a little more whitespace. `gap-4` --> `gap-6` or so